source/accessible-media/subtitles.html.erb.md

```diff
@@ -44,33 +29,17 @@ Captions 1.0 (IMSC1)](https://www.w3.org/TR/ttml-imsc1/). Keep in mind that bitm
 ## Differences between subtitles and captions
 
 Subtitles are not the same as captions, nor are they aimed at the same
-audiences. Here are some of the basic differences.
+audiences. In contrast to captions, subtitles are…
 
-<table>
-	<tr>
-		<th scope="col">Subtitles</th>
-		<th scope="col">Captions</th>
-	</tr>
-	<tr>
-		<td>Intended for people who can hear</td>
-		<td>Intended for people who are deaf or hard of hearing</td>
-	</tr><tr>
-		<td>Translation of the audio</td>
-		<td>Same language as the audio</td>
-	</tr><tr>
-		<td>Do not contain non-speech information<br>
-		(indications of music, sound effects, etc.)</td>
-		<td>Contain non-speech information</td>
-	</tr><tr>
-		<td>Not positioned to convey who is speaking</td>
-		<td>Often positioned to indicate who is speaking</td>
-	</tr>
-</table>
+* … intended for people who can hear;
+* … translations of the audio;
+* … not containing non-speech information (indications of music, sound effects, etc.);
+* … not positioned to convey who is speaking.
```

Comment:
I am not sure that we have this aspect (positioning the subtitle) covered in the captions tutorial. How would that work in a browser?